website/profile/utils.py

```diff
@@ -40,6 +40,7 @@ def serialize_user(user, node=None, admin=False, full=False):
     fullname = user.display_full_name(node=node)
     ret = {
         'id': str(user._primary_key),
+        'username': user.username,
```

Comment:
This concerns me - we've taken pains elsewhere to prevent leakage of a user's email. Is there no other way to do this?